test_data/config_test_no_optional_fields.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Nice, thanks for this test! I wonder though: do we actually have any test CVRs that push the edge case for max skipped ranks allowed being unlimited?

Reply:
Added #212 for this.